### PR TITLE
feat(docs): Add example documentation for scrollAcrossEntities

### PR DESCRIPTION
**Pull request by pedro93**
## Checklist

- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

File: docs/how/search.md [hunk 1]
```diff
@@ -179,6 +179,58 @@ for integrations and programmatic use-cases.
...
+            name
+          }
+          name
+        }
+      }
+    }
+  }
+}
+```
+This will return a response containing a `nextScrollId` value which must be used in subsequent queries to retrieve more data, i.e:
```
Comment (jjoyce0510) [inline docs/how/search.md:206]: add spacing (newlines)

File: docs/how/search.md [hunk 2]
```diff
@@ -179,6 +179,58 @@ for integrations and programmatic use-cases.
...
+          }
+          name
+        }
+      }
+    }
+  }
+}
+```
+
+This process repeats until no further `nextScrollId` are returned. Note that these subsequent operations must be performed during `keepAlive` time period which tells elasticsearch how long to keep internal scroll pointers for.
```
Comment (jjoyce0510) [inline docs/how/search.md:232]: In order to complete scrolling through all of the results, continue to request data in batches until the "nextScrollId" returned is null or undefined. 

I don't think we need to provide any detail about keepalive, but will let ryan chime in

Comment (RyanHolstien) [inline docs/how/search.md:232]: This is incorrect, until PIT is truly supported by updating our OpenSearch support to 2.x and ElasticSearch support to 8.x it is largely ignored as an option. Even when it is not ignored it is not required to query within the window however as we have logic to create a new PIT when outside of the window so there should be no issue with it. Just that you might see repeat records/missing records if there is an active ingestion.

**Review by RyanHolstien — CHANGES_REQUESTED** [review]

File: docs/how/search.md [hunk 3]
```diff
@@ -179,6 +179,58 @@ for integrations and programmatic use-cases.
 ```
 
 
+For queries that return more than 10k entities we recommend using the [scrollAcrossEntities](https://datahubproject.io/docs/graphql/queries/#scrollacrossentities):
```
Comment (jjoyce0510) [inline docs/how/search.md:182]: Can we add to a section called "Searching at Scale" 

?

Comment (pedro93) [reply docs/how/search.md:182]: Sure

File: docs/how/search.md (hunk 2 above)
Comment (pedro93) [reply docs/how/search.md:232]: What exactly is wrong? The part about keep alive?

Comment (RyanHolstien) [reply docs/how/search.md]: Yes:

`Note that these subsequent operations must be performed during `keepAlive` time period which tells elasticsearch how long to keep internal scroll pointers for.`

Comment (pedro93) [reply docs/how/search.md]: Ok, removed that. Assuming it's good now?

**Review by jjoyce0510 — APPROVED** [review]
LGTM - Thank you for this!

**Review by RyanHolstien — APPROVED** [review]

File: docs/how/search.md (hunk 2 above)
Comment (J0hnG4lt) [reply docs/how/search.md]: I've noticed that I need to retry some requests with the same scroll id until I get a response with the next scroll id. Is this behavior expected?

**Pull request merged**